### ISSUE TITLE
Nametags failed to be hidden

### ISSUE THREAD

**Issue by Xienox** [CodeCrafter47/BungeeTabListPlus#691]
Similar to issue #507, the player's nametag is still visible after drinking the potion of invisibility or given the invisibility effect. Addition to toggling f1, player nametags are still visible. The tab size is set to dynamic size and hiddenPlayers are set to invisible. There are no other plugins that interfere with nametags on both proxy and spigot servers. I never had this issue before switching from Global Tablist.

Bungeecord Build #1656
BungeeTabListPlus Version: 3.5.1

Client: Vanilla 1.8

![example](https://i.imgur.com/Sai7Rua.png)

**Comment by Villmarksjenta** [comment]
Bungeetablistplus is a tablist plugin, and does not have any effects on Nametags. You need a secondary plugin to be able to do anything with nametags.

**Comment by Xienox** [comment]
Yes, I am aware of that. However, there is something in bungeetablistplus messing with scoreboard teams. Making player names always visible no matter what you do. Even hiding the GUI with f1 player names still shows! Atop of that, I have no plugins on both spigot and proxy server affecting nametags. When removing bungeetablistplus plugin, nametags works as it was intended able to be visible or hidden.

**Comment by CodeCrafter47** [comment]
This issue is indeed caused by BungeeTabListPlus. But the bug is not in BungeeTabListPlus, but in the minecraft client. The issue has been fixed in newer versions of Minecraft.
